app/models/work.rb

```diff
@@ -149,7 +151,24 @@ def default_work(user_id, collection_id, resource, ark)
 
     work.metadata = work.resource.to_json
 
-    new_attachments = work.deposit_uploads.reject(&:persisted?)
+    new_attachments = work.pre_curation_uploads.reject(&:persisted?)
+    new_attachments.each do |attachment|
```

Comment:
Could this block be extracted to a method, so that it can be utilized for both post and pre curated attachments?